Python/instrumentation.c

```diff
@@ -111,13 +111,15 @@ static const uint8_t INSTRUMENTED_OPCODES[256] = {
 };
 
 static inline bool
-opcode_has_event(int opcode) {
+opcode_has_event(int opcode)
+{
     return opcode < INSTRUMENTED_LINE &&
         INSTRUMENTED_OPCODES[opcode] > 0;
```

Comment:
```suggestion
    return opcode < INSTRUMENTED_LINE && INSTRUMENTED_OPCODES[opcode] > 0;
```

Reply:
I'm not sure that's better.
I've gone for:
```
    return (
        opcode < INSTRUMENTED_LINE &&
        INSTRUMENTED_OPCODES[opcode] > 0
    );
```
instead.